suites/load-testing/README.md

```diff
@@ -0,0 +1,67 @@
+# k6 Load Testing
```

Comment:
let's move all this to a top level `loadTesting/` folder, and keep the `suites/` folder reserved for `codeceptjs` tests